utils/procs/patches/001-Look-for-a-config-file-in-etc.patch

```diff
@@ -0,0 +1,35 @@
+From c42678c121527a255826425e848a90ffefbe25d0 Mon Sep 17 00:00:00 2001
+From: Facundo Acevedo <facevedo@disroot.org>
+Date: Tue, 4 Jul 2023 17:26:51 -0300
+Subject: [PATCH] Look for a config file in etc
```

Comment:
Is this patch going to be upstreamed? If not, then you will need to rebase it/refresh it each time if there is a new release and there were changes in ``src/main.rs``.